go-controller/hybrid-overlay/pkg/controller/node_linux_test.go

```diff
@@ -40,16 +36,78 @@ const (
 	testNodeIP     string = "1.2.3.3"
 )
 
+// returns if the two flowCaches are the same
+func compareFlowCache(returnedFlowCache, expectedFlowCache map[string]*flowCacheEntry) bool {
+	if len(returnedFlowCache) != len(expectedFlowCache) {
+		for key, entry := range returnedFlowCache {
+			fmt.Printf("KEYWORD: key %s\n", key)
+			for _, flow := range entry.flows {
+				fmt.Printf("KEYWORD: flow %s\n", flow)
+			}
+		}
+		return false
+	}
+
+	for key, entry := range returnedFlowCache {
+		expectedEntry, ok := expectedFlowCache[key]
+		if !ok {
+			return false
```

Comment:
earlier in this function you print if the cache lengths dont match, do you want to print the flow that was not expected here? Should they be klog.Errors?

Reply:
the printing was actually code I was using to debug and left in, now that I took it out do you still think I should log the unexpected flows?

Reply:
well I was just thinking if the test fails it would be good to have what is missing logged so someone can debug easier. wdyt?

Reply:
I added a klog.Error